common/culture/traditions/00_combat_traditions.txt

```diff
@@ -2060,14 +2137,15 @@ tradition_horse_breeder = {
 		1 = mena
 		4 = horses.dds
 	}
-
-	is_shown = {
-		OR = {
-			culture_has_light_cavalry_maa = yes
-			culture_has_heavy_cavalry_maa = yes
-			culture_has_archer_cavalry_maa = yes
-		}
-	}
+	
+	# Warcraft
+	# is_shown = {
+		# OR = {
+		#	culture_has_light_cavalry_maa = yes
+		#	culture_has_heavy_cavalry_maa = yes
+		#	culture_has_archer_cavalry_maa = yes
+		# }
+	# }
```

Comment:
Already exists in triggers

Reply:
These triggers are defined in `00_cultural_triggers.txt` and existed in vanilla for MaA traditions. However because the relevant vanilla MaAs are not included in our mod, the triggers are checked for no purpose as of the moment. Would you prefer repurposing these triggers or creating a overarching new wc trigger `has_no_cavalry_culture`?

Reply:
> creating a overarching new wc trigger `has_no_cavalry_culture`?

I would. `has_cavalry_culture`, negative triggers can be confusing.